WordPress/Classes/Services/LoginFacade.m

```diff
@@ -68,17 +71,28 @@ - (void)signInToSelfHosted:(LoginFields *)loginFields
             if ([options objectForKey:@"wordpress.com"] != nil) {
                 [self signInToWordpressDotCom:loginFields];
             } else {
+                NSString *versionString = options[@"software_version"][@"value"];
```

Comment:
@astralbodies  I agree that the BlogService code should be removed... but we have some scenarios to think about:

1. What happens for users that already added sites that are older than the current minimum version? We just let them be? Show a warning on startup of the app and log them out?
2. Even with this code I can add a site today, a 6 months from now if we push the minimum version up what do we do for that site?

Should I merge this PR and create a new ticket for these issues?